cwms-data-api/src/test/java/cwms/cda/api/TimeseriesControllerTestIT.java

```diff
@@ -1,32 +1,709 @@
 package cwms.cda.api;
 
-import java.sql.SQLException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
 import javax.servlet.http.HttpServletResponse;
 
-import com.fasterxml.jackson.databind.JsonNode;
-import com.fasterxml.jackson.databind.ObjectMapper;
-
 import cwms.cda.formatters.Formats;
 import fixtures.TestAccounts;
-import fixtures.TestAccounts.KeyUser;
-import io.restassured.RestAssured;
+import io.jsonwebtoken.io.IOException;
 import io.restassured.filter.log.LogDetail;
-import io.restassured.path.json.config.JsonPathConfig;
 import org.apache.commons.io.IOUtils;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.Test;
 
 import static io.restassured.RestAssured.given;
-import static io.restassured.config.JsonConfig.jsonConfig;
-import static org.hamcrest.Matchers.closeTo;
-import static org.hamcrest.Matchers.contains;
-import static org.hamcrest.Matchers.is;
-import static org.hamcrest.Matchers.nullValue;
+import static org.hamcrest.Matchers.*;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
 
 @Tag("integration")
 public class TimeseriesControllerTestIT extends DataApiTestIT {
 
+    private static final String OFFICE = "SPK";
+    private static final String locationId = "TsVersionedTestLoc";
+    private static final String tsId = locationId + ".Flow.Inst.1Hour.0.raw";
+    public static final String BEGIN_STR = "2008-05-01T15:00:00Z";
+    public static final String END_STR = "2008-05-01T17:00:00Z";
+
+    @BeforeAll
+    public static void create() throws Exception {
+        createLocation(locationId, true, OFFICE);
+        createTimeseries(OFFICE, tsId, 0);
+    }
+
+    @Test
+    void test_create_get() throws IOException, java.io.IOException {
```

Comment:
method name should specify this is testing for date versioned ts. Also you can just throw `Exception` from tests. The two `IOException`'s are confusing

Reply:
Done